addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.0\n"
+"Project-Id-Version: Odoo Server 14.0+e\n"
```

Comment:
@mart-e I just noticed this. Is this okay or should I remove this '+e'.

Reply:
It's no big deal but as you have noticed, it's good to keep community versions in community modules

Reply:
Got it. Thanks.